src/format-files.test.js

```diff
@@ -116,7 +116,11 @@ test('logs errors to the console if something goes wrong', async () => {
   const errorPrefix = expect.stringMatching(/prettier-eslint-cli.*ERROR/)
   const cliError = expect.stringContaining('eslint-config-error')
   const errorOutput = expect.stringContaining('Some weird eslint config error')
-  expect(console.error).toHaveBeenCalledWith(errorPrefix, cliError, errorOutput)
+  expect(console.error).toHaveBeenCalledWith(
```

Comment:
The prettier-eslint git hook. My guess is prettier was updated since the last time this code was formatted. No worries. I don't mind unrelated formatting changes because prettier's doing it and I <3 prettier :)